scripts/travis/build.sh

```diff
@@ -97,7 +97,7 @@ else
 fi
 
 # Build the current project
-mvn -B -V -q install -DskipTests -Dmaven.javadoc.skip=true -Pbuild-docker-images -Pags ${REPO_IMAGE} ${SHARE_IMAGE}
+mvn -B -V install -DskipTests -Dmaven.javadoc.skip=true -Pbuild-docker-images -Pags ${REPO_IMAGE} ${SHARE_IMAGE}
```

Comment:
Is this change desired?  I can see it's useful for debugging, but does it bloat the log?  Previously we've had success by enabling this, but turning off the logging around "Downloading..." and "Downloaded..." messages - perhaps that would be a compromise?

Reply:
was for debugging, will be removed  by rebase